e2e/journeys/citizen/createCase/C100OtherProceedings/C100OtherProceedings.ts

```diff
@@ -66,6 +75,9 @@ export class C100OtherProceedings {
     yesNoOccupationOrderDetails,
     yesNoForcedMarriageProtectionOrderDetails,
     yesNoRestrainingOrderDetails,
+    yesNoOtherInjunctionOrderDetails,
+    yesNoUndertakingOrderDetailsOrderDetails,
+    yesNoOtherOrderDetailsDetails,
```

Comment:
Can you make a single argument for the whole subJourney: `c100OtherProceedingsYesNoToAll` so it simplifies the number of variables? We're only going to be doing yes to all or no to all in the tests.

Reply:
Was planning to do that in a different PR

Reply:
Is this the last OP ticket? I'm happy to make this change in the stitching effort if so

Reply:
No, I have to do the summary page for it. I was planning on doing it in that ticket